logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -278,14 +281,15 @@ def join(self, timeout=None):
 
 
 class _Batch(object):
+    from google.cloud.logging.logger import _GLOBAL_RESOURCE
```

Comment:
This import breaks the "import at run-time" philosophy, because this class (and it's attrs) are defined at import time.